src/pages/STG_PanelAddrVerification.page

```diff
@@ -196,38 +187,39 @@
             </apex:outputPanel>
         </apex:outputPanel>
         </div>
-        
-        <br/>
-            <section>
-                <div id="iBatchVerifyDiv" class="col-sm-offset-1 col-sm-10">
-                    <div class="panel panel-default">
-                        <div class="panel-heading">
-                            <h3 class="panel-title">{!$Label.Addr_Verification_Batch_Title}</h3>
-                        </div>
-                        <div class="panel-body" style="padding:15px;">                 
-                            <apex:outputText escape="false" value="{!$Label.Addr_Verification_Batch_Body}" />
-                            <apex:outputText id="validatorBatchHelpText" escape="false" value="{!validatorBatchHelpText}" />
-                            <label class="checkbox" style="margin-left: 15px"> 
-                                    <apex:inputCheckbox value="{!skipPreviouslyVerifiedObjects}" />{!$Label.Addr_Skip_Verified}
-                            </label>
-                            <div style="text-align:center; margin-top: 15px;">
-                                <apex:commandButton id="batchVerify" action="{!batchVerify}" rerender="batchVerifyResult" value="Verify All Addresses" disabled="{!NOT batchVerifEnabled}" styleClass="btn-sm btn-default"/> 
+
+        <div class="slds-m-around--large" id="iBatchVerifyDiv">
```

Comment:
The Mass Verify Existing Addresses box only displays when you pick Smarty Streets, but the verify button is disabled, and there is a message that the selected service does not support Batch Address Validation.